cmd/root.go

```diff
@@ -59,7 +59,7 @@ const (
 	RemotePrefix = "remote://"
 )
 
-var layerAnalyzers = [...]string{"layer", "aptlayer"}
+var layerAnalyzers = [...]string{"layer", "aptlayer", "rpmlayer"}
```

Comment:
since you're in here, would you mind making these strings constants somewhere?

Reply:
I will include the constants somewhere